src/main/java/org/spongepowered/api/data/manipulator/mutable/entity/InvisibilityData.java

```diff
@@ -40,16 +41,26 @@
      *
      * <p>Note that this is different from the {@link #vanish()} state as when an
      * {@link Entity} is "invisible", update packets are still sent to all clients
-     * and the server. Likewise, no </p>
+     * and the server.</p>
      *
      * @return The value of the invisible state
+     * @see Keys#INVISIBLE
      */
     Value<Boolean> invisible();
 
     /**
      * Gets the {@link Value} of the "vanish" state of an {@link Entity}.
      *
+     * <p>The presence of a vanished entity will not be made known to a client;
+     * no packets pertaining to this entity are sent. Client-side, this entity
+     * will cease to exist. Server-side it may still be targeted by hostile
+     * entities or collide with other entities.</p>
```

Comment:
Make note of the other settings, as this does also affect whether other settings will apply (ignores collision requires the entity is vanished in the first place).